auth-web/tests/unit/views/InvoluntaryDissolution.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { Wrapper, createLocalVue, mount } from '@vue/test-utils'
```

Comment:
Preliminary test of the view. We're going to probably add more stuff in the future when the actual view has been implemented.